src/services/Look/config.ts

```diff
@@ -0,0 +1,20 @@
+import { IncomingMessage } from 'http';
+
+import { baseConfig } from '../Api';
+
+import { ILookInputDTO } from '@src/interfaces';
+
+export const listConfig = (req?: IncomingMessage) =>
+  baseConfig(true, req).get('/super/looks/', {});
+
+export const partialUpdateConfig = (
+  id: number,
+  updateDTO: ILookInputDTO,
```

Comment:
```suggestion
  updateDTO: Partial<ILook>,
```

update할땐 일부 property만 전달할 수 있으므로 Partial을 사용해야합니다!

Reply:
f399f14